babel.config.js

```diff
@@ -21,7 +25,7 @@ module.exports = function (api) {
   ]
 
   const plugins = [
-    "@chakra-ui/babel-plugin",
+    transformPureCalls,
```

Comment:
`@chakra-ui/babel-plugin` was causing issues with build and local dev. 

If I added it as a `preconstruct` package, build would fail because it wasn't necessarily built before the other packages since it isn't technically a dependency of anything else.

If I didn't add it, we'd need to guarantee that it was built before `preconstruct dev` was run, which meant adding it to `postinstall`, which meant a slightly annoying little delay anytime you installed modules.

So rather than add it as a dependency to a bunch of things or add it to `postinstall`... I deleted the whole package and inlined it. 😲